tests/mdev_configuration_allocation_test.go

```diff
@@ -118,14 +118,17 @@ var _ = Describe("[Serial][sig-compute]MediatedDevices", func() {
 		var desiredMdevTypeName string = "nvidia-222"
 		var expectedInstancesNum int = 16
 		var config v1.KubeVirtConfiguration
+		var originalConfig v1.KubeVirtConfiguration
 		var mdevTestLabel = "mdevTestLabel1"
 
 		BeforeEach(func() {
 			kv := util.GetCurrentKv(virtClient)
 
 			By("Creating a configuration for mediated devices")
 			config = kv.Spec.Configuration
-			config.DeveloperConfiguration.FeatureGates = []string{virtconfig.GPUGate}
+			originalConfig = *config.DeepCopy()
+			DeferCleanup(func() { tests.UpdateKubeVirtConfigValueAndWait(originalConfig) })
```

Comment:
I think this needs to come after the actual change (at line 147), as the change itself may fail and there is nothing to revert.